biome.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+    "vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": false },
+    "files": {
+        "ignoreUnknown": true,
+        "ignore": ["services/**/*.mjs", "node_modules", "*/**/node_modules", ".sst", ".sst*", "*/**/botocore"]
```

Comment:
Is Biome unable to process *.mjs files?

Reply:
Looks like biome can process .mjs files - I've updated the config file. Think it might have been included when running the migration script from eslint